### PR TITLE
feat: add SDK supporting arbitrum

### PR DESCRIPTION
**Pull request by anxolin**
# Description
Add arbitrum

### CODE REVIEW

**Comment by alfetopito** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by alfetopito** [comment]
This is running on watchtower infra and is working as expected.

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
   "dependencies": {
     "@commander-js/extra-typings": "^11.0.0",
     "@cowprotocol/contracts": "^1.4.0",
-    "@cowprotocol/cow-sdk": "^5.2.0",
+    "@cowprotocol/cow-sdk": "^5.3.0-RC.2",
```
Comment (mfw78) [inline package.json:49]: Is the intent to keep this as a release candidate, or is the intent to bump this to 5.3.0 when it's final?

**Review by alfetopito — APPROVED** [review]

**Review by mfw78 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (anxolin) [reply package.json]: Yes, lets do it

Comment (anxolin) [reply package.json]: Okay, @alfetopito  I see you went already for it https://www.npmjs.com/package/@cowprotocol/cow-sdk?activeTab=versions

Comment (alfetopito) [reply package.json]: Yes, sorry forgot to comment in this thread

**Pull request merged**